src/Home.js

```diff
@@ -0,0 +1,351 @@
+      default:
+        return;
+    }
+  }
+  const getGridRow = (date) => {
+    switch (date.toLocaleString('default', {hour: '2-digit', minute: '2-digit'})) {
+      case '07:00 AM':
+        return 'AM7start';
+      case '07:30 AM':
+        return 'AM730start';
+      case '08:00 AM':
+        return 'AM8start';
+      case '08:30 AM':
+        return 'AM830start';
+      case '09:00 AM':
+        return 'AM9start';
+      case '09:30 AM':
+        return 'AM930start';
+      case '10:00 AM':
+        return 'AM10start';
+      case '10:30 AM':
+        return 'AM1030start';
+      case '11:00 AM':
+        return 'AM11start';
+      case '11:30 AM':
+        return 'AM1130start';
+      case '12:00 PM':
+        return 'PM12start';
+      case '12:30 PM':
+        return 'PM1230start';
+      case '01:00 PM':
+        return 'PM1start';
+      case '01:30 PM':
+        return 'PM130start';
+      case '02:00 PM':
+        return 'PM2start';
+      case '02:30 PM':
+        return 'PM230start';
+      case '03:00 PM':
+        return 'PM3start';
+      case '03:30 PM':
+        return 'PM330start';
+      case '04:00 PM':
+        return 'PM4start';
+      case '04:30 PM':
+        return 'PM430start';
+      case '05:00 PM':
+        return 'PM5start';
+      case '05:30 PM':
+        return 'PM530start';
+      case '06:00 PM':
+        return 'PM6start';
+      case '06:30 PM':
+        return 'PM630start';
+      case '07:00 PM':
+        return 'PM7start';
+      case '07:30 PM':
+        return 'PM730start';
+      case '08:00 PM':
+        return 'PM8start';
+      case '08:30 PM':
+        return 'PM830start';
+      case '09:00 PM':
+        return 'PM9start';
+      case '09:30 PM':
+        return 'PM930start';
+      case '10:00 PM':
+        return 'PM10start';
+      default:
+        return;
+    }
+  }
+  
+  // contains the displayed list of events
+  const CalendarEvents = () => {
+    const gridLines = [];
+  
+    // Background Grid
+    for (let row = 1; row <= getCSSVar('--gridRowCount'); row++) {
+      for (let col = 1; col <= getCSSVar('--gridColCount'); col++) {
+        gridLines.push(
+          <div className='gridLine' key={`${row}-${col}`} style={{gridRow: row, gridColumn: col}} div/>
+        )
+      }
+    }
+  
+    return (
+      <section className='calendarGrid'>
+        {EVENTS.map((event) => {
+            return (
+              <Event name={event.name} 
+                    start={event.start} 
+                    end={event.end} 
+                    location={event.location} 
+                    sideColor={event.sideColor} 
+                    bgColor={event.bgColor}
+                    otherParty={event.otherParty}
+                    initials1={event.initials1}
+                    iconColor1={event.iconColor1}
+              />
+            )
+          })}
+        {gridLines}
+      </section>
+    )
+  }
+  // defines how an event is displayed
+  function Event(props) {
```

Comment:
You should define the Event component before the CalendarEvents component because CalendarEvents uses Event. By convention, components are defined before they are used.